unit_test/tstIntegrator.hpp

```diff
@@ -35,7 +35,12 @@ t_System createParticles( const int num_particle, const int num_ghost,
     // Manually setup what would be done in input
     system.dt = 0.005;
     system.mvv2e = 1.0;
-    system.mass( 0 ) = 1.0;
+
+    // Set mass (device View)
+    using h_t_mass = typename t_System::h_t_mass;
+    h_t_mass h_mass = Kokkos::create_mirror_view( system.mass );
+    h_mass( 0 ) = 1.0;
+    Kokkos::deep_copy( system.mass, h_mass );
```

Comment:
You could of course also just do
```
Kokkos::deep_copy( system.mass, 1.0 );
```
if you don't mind copying the same scalar into all elements.

Reply:
That is indeed much simpler.